@xen-orchestra/backups/writers/_AbstractBackupWriter.js

```diff
@@ -0,0 +1,29 @@
+const { asyncMap } = require('@xen-orchestra/async-map')
+const { createLogger } = require('@xen-orchestra/log')
+
+const { BACKUP_DIR } = require('../_getVmBackupDir')
+
+const { debug } = createLogger('xo:backups:AbstractBackupWriter')
+
+exports.AbstractBackupWriter = (Sub = Object) =>
+  class AbstractBackupWriter extends Sup {
```

Comment:
Do you think `Sub` is clear enough? What does it mean?

```suggestion
exports.AbstractBackupWriter = (BaseClass = Object) =>
  class AbstractBackupWriter extends BaseClass {
```

Reply:
It's `sup` not `sub`. Which means superclass

Reply:
My bad, I introduced the typo myself :sweat_smile: